docs/guides/grafana-vmgateway-openid-configuration.md

```diff
@@ -0,0 +1,162 @@
+# How to configure vmgateway for multi-tenant access using Grafana and OpenID Connect
+
+Using Grafana with vmgateway is a great way to provide multi-tenant access to your metrics.
+vmgateway provides a way to authenticate users using JWT tokens issued by an external identity provider.
+Those tokens can include information about the user and the tenant they belong to, which can be used
+to restrict access to metrics to only those that belong to the tenant.
+
+## Prerequisites
+
+* Identity service that can issue JWT tokens
+* Grafana
+* VMSingle or VMCluster
+* vmgateway
+
+## Configure identity service
+
+The identity service must be able to issue JWT tokens with the following `vm_access` claim. Here is an example claims
+object:
+
+```json
+{
+  "vm_access": {
+    "tenant_id": {
+      "account_id": 0,
+      "project_id": 0
```

Comment:
can project_id be omitted?

Reply:
Yes, it will be set to 0 in case it is omitted. Would it be better to remove it?

Reply:
Added a note for that